frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -242,6 +266,11 @@ export const OperatorHubItemDetails: React.FC<OperatorHubItemDetailsProps> = ({
                   )
                 }
               />
+              { (isTokenEnabledCluster(cloudcredentials, infrastructure, authentication)) ?
+              <PropertyItem
+                label={t('olm~Short Lived Token Enabled')}
```

Comment:
Check with UXD on casing.  I think this should be
```suggestion
                label={t('olm~Short lived token enabled')}
```